### DISCUSSION TITLE
Error while Installing

### DISCUSSION THREAD

**Discussion by AliMohamedco** [Security-Onion-Solutions/securityonion#13752] [Unsupported Versions]
I had an unknown error while installing the Standalone version could some me share ideas.
Never had any luck with getting Security Onion to work sadly.
![error 1](https://github.com/user-attachments/assets/5dbb00f7-5166-4204-82d8-4c2ff148fefe)
![error 2](https://github.com/user-attachments/assets/3046dc3c-c5d3-4c29-b446-82da312cf071)
![error 3](https://github.com/user-attachments/assets/8ec3bef5-0594-48f2-811d-1299d4978e41)
![error 4](https://github.com/user-attachments/assets/14bb5d0f-a5d5-48b6-b4db-389d5bcbfac4)
![error 5](https://github.com/user-attachments/assets/b1a64aca-bd80-4881-826e-2f679950d262)
![image](https://github.com/user-attachments/assets/a223656d-48d9-4ecf-83cb-c668f63a2c2a)

**Comment by rh-ops** [comment]
What happens if you run "sudo so-checkin"? Does it complete without error? Also, what does "sudo nmcli" con show return?

**Reply by AliMohamedco** [reply]
![1](https://github.com/user-attachments/assets/c629147a-418c-4f34-8bda-c9457fb15b6e)
![2](https://github.com/user-attachments/assets/ac8bf011-2832-4a2e-b44a-c91fe08b0465)
![3](https://github.com/user-attachments/assets/f90a9e78-77a5-4913-aa52-7bc7ce8e48d6)

**Reply by rh-ops** [reply]
When you run "ip link show ens34", what is the status of the interface? Make sure it isnt marked as "DOWN", if it is down, bring it back up using the command "sudo ip link set ens34 up". Check if the interface has an ip address with the following command "ip addr show ens34".
